x-pack/plugins/ml/public/application/components/custom_urls/custom_url_editor/editor.tsx

```diff
@@ -29,10 +30,11 @@ import { DataView } from '@kbn/data-views-plugin/public';
 import { CustomUrlSettings, isValidCustomUrlSettingsTimeRange } from './utils';
 import { isValidLabel } from '../../../util/custom_url_utils';
 import { type DataFrameAnalyticsConfig } from '../../../../../common/types/data_frame_analytics';
-import { Job, isAnomalyDetectionJob } from '../../../../../common/types/anomaly_detection_jobs';
+import { Job } from '../../../../../common/types/anomaly_detection_jobs';
```

Comment:
```suggestion
import { type Job } from '../../../../../common/types/anomaly_detection_jobs';
```

Reply:
Updated in eca357ab60ac4e8d248f0fa54906aab332d5c42e